src/store/app.ts

```diff
@@ -55,6 +55,8 @@ export const networkConfigAtom = atomWithStorage(
   networkConfigs.mainnet
 );
 
+export const termsAgreedAtom = atomWithStorage('terms-agreed', -1);
```

Comment:
elsewhere this reads like a boolean. consider `termsAgreedUponAtom`

Reply:
changed to `termsIndexAgreedUponAtom`